Framework/API/src/ScriptBuilder.cpp

```diff
@@ -177,8 +177,8 @@ const std::string ScriptBuilder::buildAlgorithmString(const AlgorithmHistory &al
     Mantid::API::Algorithm_sptr algFresh;
     // create a fresh version of the algorithm - unmanaged
     if (name == "Load") {
-      algFresh =
-          AlgorithmManager::Instance().createUnmanaged(algHistory.getPropertyValue("LoaderName"), algHistory.version());
+      int version = atoi(algHistory.getPropertyValue("LoaderVersion").c_str());
```

Comment:
Can we use the [C++ version](https://en.cppreference.com/w/cpp/string/basic_string/stol) of this instead as it just accepts a `std::string`:
```
const int version = std::stoi(algHistory.getPropertyValue("LoaderVersion"))
```